assets/app/view/game/part/city_slot.rb

```diff
@@ -23,6 +23,7 @@ class CitySlot < Base
         needs :reservation, default: nil
         needs :game, default: nil, store: true
         needs :city_render_location, default: nil
+        needs :player_colors, default: nil
```

Comment:
why don't you just make this a store, then you don't have to pass it down everywhere it's unnneded?

Reply:
hopefully a bit simplier